package.json

```diff
@@ -6,7 +6,9 @@
   },
   "scripts": {
     "prepare": "husky install",
-    "sync_with_wheel": "curl -s -L \"https://raw.githubusercontent.com/bigbinary/wheel/master/.scripts/sync_with_wheel.sh\" | bash"
+    "sync_with_wheel": "curl -s -L \"https://raw.githubusercontent.com/bigbinary/wheel/master/.scripts/sync_with_wheel.sh\" | bash",
+    "lint": "eslint \"app/javascript/src/**/*.{js,jsx}\"",
+    "lint:fix": "eslint --fix \"app/javascript/src/**/*.{js,jsx}\""
```

Comment:
Raise an issue in https://github.com/bigbinary/wheel to add this there and self-assign yourself and make a PR for it. You can assign me the PR.

The linting should also have `prettier` check, so should the fix command.